mm2src/coins/z_coin.rs

```diff
@@ -1,42 +1,40 @@
 use crate::coin_errors::MyAddressError;
+#[cfg(not(target_arch = "wasm32"))]
 use crate::my_tx_history_v2::{MyTxHistoryErrorV2, MyTxHistoryRequestV2, MyTxHistoryResponseV2};
+#[cfg(not(target_arch = "wasm32"))]
```

Comment:
Why are `#[cfg(not(target_arch = "wasm32"))]` needed here? we are supposed to remove those too, or are you leaving those to know the parts that need implementations in the future?

Reply:
I added #[cfg(not(target_arch = "wasm32"))]` to every storage function as they're all currently implemented for sql.. I'll implemented them one by one for `indexeddb`